src/hdf5_time_series_storage.jl

```diff
@@ -85,11 +85,22 @@ end
 
 get_file_path(storage::Hdf5TimeSeriesStorage) = storage.file_path
 
+function _time_array_wrapper_to_array(ta::TimeDataContainer)
```

Comment:
This should live in the file containing TimeDataContainer.  And it needs a new name.

Reply:
ok, I will move it